operatorcourier/verified_manifest.py

```diff
@@ -34,6 +35,14 @@ def __init__(self, source_dir, yamls, ui_validate_io, repository):
             self.get_validation_dict_from_manifests(manifests, ui_validate_io, repository)
         self.is_valid = False if self.__validation_dict['errors'] else True
 
+    def _set_empty_filepaths(self, yamls):
```

Comment:
Could you explain what this function does?

Reply:
When you call the api with `yamls` set instead of a file path there are no file names. So this function just creates the tuples for the file list and sets them all to empty string.